content/rest/reference/repos.md

```diff
@@ -144,7 +144,7 @@ Note that the `repo_deployment` [OAuth scope](/developers/apps/scopes-for-oauth-
 
 ### Inactive deployments
 
-When you set the state of a deployment to `success`, then all prior non-transient, non-production environment deployments in the same repository will become `inactive`. To avoid this, you can set `auto_inactive` to `false` when creating the deployment status.
+When you set the state of a deployment to `success`, then all prior non-transient, non-production environment deployments in the same repository to the same environment (based on the value of environment) will become `inactive`. To avoid this, you can set `auto_inactive` to `false` when creating the deployment status.
```

Comment:
optional wording suggestion:

```suggestion
When you set the state of a deployment to `success`, then all prior non-transient, non-production environment deployments in the same repository to the same environment name will become `inactive`. To avoid this, you can set `auto_inactive` to `false` when creating the deployment status.
```

Reply:
@skedwards88 - I like that suggestion!  I committed it.  Thanks.